analysis/plotting.py

```diff
@@ -13,12 +13,12 @@
 # limitations under the License.
 """Plotting functions."""
 
-import matplotlib.pyplot as plt
-import matplotlib.colors as colors
 import numpy as np
 import Orange
 import seaborn as sns
 
+from matplotlib import colors
```

Comment:
Did google's styleguide get rid of alphabetical order imports? It looks like it might have.

Reply:
Yep, I remember `pylint` specifically requested me to put this at the top (which also surprised me), because it is an external import.